src/components/Records/Record/Keywords.vue

```diff
@@ -24,7 +24,8 @@
           outlined
           text-color="primary"
         >
-          <v-icon left small>
+          <v-icon left
```

Comment:
Why did this file change? It created four minor styling issues that need fixing before i can accept the PR.
Please see:
https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=6425475